src/vstest.console/Internal/ConsoleLogger.cs

```diff
@@ -765,7 +765,7 @@ private void TestRunCompleteHandler(object? sender, TestRunCompleteEventArgs e)
                     : $"({_targetFramework})";
 
                 var duration = GetFormattedDurationString(sourceSummary.Duration);
-                var sourceName = sd.Key.Split('\\').Last();
+                var sourceName = sd.Key.Split(Path.DirectorySeparatorChar, Path.AltDirectorySeparatorChar).Last();
```

Comment:
Could `Path.GetFileName` be used here? It handles both normal and alt separator char. But does not handle UNC paths well. So maybe this is a lower risk option.

It still seems like a one-off solution of something that should be in a "library" and used everywhere. That is different discussion though.

Reply:
> But does not handle UNC paths well

Do you know specific cases that are not well handled? Because replacing the code with `Path.GetFileName` seems pretty sensible to me.